diff --git a/dist/serverless/adapter.js b/dist/serverless/adapter.js
index ae8cd14edeb2c46c6ef4b3563c4f32851f3f0477..e81d567f2e940fbcea0a831a94c0d0e2ac65b7cf 100644
--- a/dist/serverless/adapter.js
+++ b/dist/serverless/adapter.js
@@ -35,7 +35,8 @@ const ISR_PATH = `/_isr?${ASTRO_PATH_PARAM}=$0`;
 // https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#node.js-version
 const SUPPORTED_NODE_VERSIONS = {
     18: { status: 'retiring', removal: 'Early 2025', warnDate: new Date('October 1 2024') },
-    20: { status: 'default' },
+    20: { status: 'retiring', removal: 'Late 2025', warnDate: new Date('June 1 2025') },
+    22: { status: 'default' },
 };
 function getAdapter({ edgeMiddleware, functionPerRoute, middlewareSecret, skewProtection, }) {
     return {
